source/tests/system/nirfmxnr_session_tests.cpp

```diff
@@ -0,0 +1,117 @@
+#include <gtest/gtest.h>
```

Comment:
Note: Since this was so similar to the `nirfmxspecan_session_tests` I figured I'd give these the same versions to get that coverage. I also wanted to have the `driver_api_tests` set up as well for future System Level tests so I went ahead and added it with a simple `Init_Close_Succeeds` test. I can delete the Initialize test in the `driver_api_tests` when we add more, but thought it'd be good to have at least one test in that style up front to have the `experimental::client` framework validated for NR.